### PR TITLE
Add Travis CI configuration.

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,7 +10,8 @@ watch:
 
 init:
 	brew update
-	brew install hugo
+	brew install hugo pre-commit
```
Comment (rumyantseva) [inline Makefile:13]: What if someone doesn't want to install `pre-commit` tool?

Comment (AlekSi) [reply Makefile:13]: Then that someone can run those commands by hands, nothing prevents they from doing that. 😄

Comment (rumyantseva) [reply Makefile:13]: Ok

**Comment by AlekSi** [comment]
PTAL

**Review by rumyantseva — APPROVED** [review]
LGTM

**Pull request merged**